### PR TITLE
Issue #461 Fix: Project details showing No Results in textbooks list for first few seconds

### PR DESCRIPTION
**Pull request by niteshkesarkar**

### CODE REVIEW

File: src/app/client/src/app/modules/cbse-program/components/collection/collection.component.html [hunk 1]
```diff
@@ -111,97 +111,102 @@ <h5 class="m-0 p-0 font-weight-bold d-inline-block">{{programContext.name}} </h5
 
   <div class="sb-bg-color-white py-24 dynamicHeight">
     <div class="ui container m-0 p-0 sb-pageSection">
-      <div class="sb-pageSection-header mb-15">
-        <div>
-          <h4 class="sb-pageSection-title six">{{resourceService.frmelmnts.lbl.textbooks}}</h4>
-          <span class="sb-pagSection-count sb-label sb-label-x sb-label-warning mx-10">{{collectionList.length || ''}}</span>
+      <ng-container *ngIf="!showLoaderTexbooks">
```
Comment (twsvaishali) [inline src/app/client/src/app/modules/cbse-program/components/collection/collection.component.html:114]: @niteshkesarkar only use showLoader var

File: src/app/client/src/app/modules/cbse-program/components/collection/collection.component.html [hunk 2]
```diff
@@ -111,97 +111,102 @@ <h5 class="m-0 p-0 font-weight-bold d-inline-block">{{programContext.name}} </h5
...
+                <th class="w-5 cursor-pointer sb-sorting" (click)="sortCollection('subject')">
+                  {{resourceService.frmelmnts.lbl.subjects}} &nbsp;&nbsp;
+                  <i class="fa fa-sort" aria-hidden="true" *ngIf="direction == '' || sortColumn != 'subject' "></i>
+                  <span  *ngIf="sortColumn == 'subject'" class="d-inline-block">
+                    <i class="fa fa-sort-asc" aria-hidden="true" [ngClass]="{'sb-color-gray-100':direction == 'asc'}"></i>
+                    <i class="fa fa-sort-desc" aria-hidden="true" [ngClass]="{'sb-color-gray-100':direction == 'desc'}"></i>
+                  </span>
+                </th>
+                <th class="w-5">{{resourceService.frmelmnts.lbl.chapters}}</th>
+                <th class="w-5" *ngIf="currentNominationStatus === 'Initiated' || currentNominationStatus=== 'Pending'">Sample Content</th>
```
Comment (twsvaishali) [inline src/app/client/src/app/modules/cbse-program/components/collection/collection.component.html:163]: lang constant?

File: src/app/client/src/app/modules/cbse-program/components/collection/collection.component.html (hunk 1 above)
Comment (niteshkesarkar) [reply src/app/client/src/app/modules/cbse-program/components/collection/collection.component.html]: @twsvaishali - Yes, used `showLoader` now.

File: src/app/client/src/app/modules/cbse-program/components/collection/collection.component.html (hunk 2 above)
Comment (niteshkesarkar) [reply src/app/client/src/app/modules/cbse-program/components/collection/collection.component.html]: @twsvaishali - Added missing language constants.

File: src/app/client/src/app/modules/contribute/components/list-nominated-textbooks/list-nominated-textbooks.component.ts [hunk 3]
```diff
@@ -209,12 +210,14 @@ export class ListNominatedTextbooksComponent implements OnInit, AfterViewInit, O
           }
 
           this.tempSortTextbooks = this.contributorTextbooks;
+          this.showLoaderTexbooks = false;
```
Comment (twsvaishali) [inline src/app/client/src/app/modules/contribute/components/list-nominated-textbooks/list-nominated-textbooks.component.ts:213]: showLoaderTexbooks = showLoader

Comment (niteshkesarkar) [reply src/app/client/src/app/modules/contribute/components/list-nominated-textbooks/list-nominated-textbooks.component.ts]: @twsvaishali  -Updated.

**Pull request merged**